test/extended/util/framework.go

```diff
@@ -1709,7 +1718,7 @@ func (r *podExecutor) Exec(script string) (string, error) {
 	var out string
 	waitErr := wait.PollImmediate(1*time.Second, 3*time.Minute, func() (bool, error) {
 		var err error
-		out, err = r.client.Run("exec").Args(r.podName, "--", "/bin/bash", "-c", script).Output()
+		out, err = r.client.AsAdmin().Run("exec").Args(r.podName, "--", "/bin/bash", "-c", script).Output()
```

Comment:
why did this change?

Reply:
So I remember doing this for `"[sig-imageregistry][Serial] Image signature workflow can push a signed image to openshift registry and verify it [apigroup:user.openshift.io][apigroup:image.openshift.io] [Skipped:Disconnected] [Suite:openshift/conformance/serial]"`, regarding https://github.com/openshift/origin/pull/27635#issuecomment-1397075094,  but I tested again without `AsAdmin()` in my own cluster, and it worked. So I'm confused now. I thought it was causing the permission denied issues, but that doesn't really make sense to me anymore why `AsAdmin()` would impact that.

It's possible I had two changes in flight and mixed them up. Let me remove and test in CI to confirm we don't need it.